airflow/migrations/versions/0140_2_9_0_update_trigger_kwargs_type.py

```diff
@@ -45,26 +45,10 @@ def get_session() -> sa.orm.Session:
 
 
 def upgrade():
-    """Apply encrypt trigger kwargs"""
-    session = get_session()
-    try:
-        op.alter_column(table_name="trigger", column_name="kwargs", type_=sa.String())
-        for trigger in session.query(Trigger).all():
-            # convert dict to string and encrypt it
-            trigger.encrypted_kwargs = trigger._encrypt_kwargs(trigger.encrypted_kwargs)
-        session.commit()
-    finally:
-        session.close()
+    """Update trigger kwargs type to string"""
+    op.alter_column(table_name="trigger", column_name="kwargs", type_=sa.String())
```

Comment:
```suggestion
    op.alter_column(table_name="trigger", column_name="kwargs", type_=sa.Text())
```
I think `Text` is in the ORM?